src/renderer/components/+extensions/extensions.tsx

```diff
@@ -91,11 +91,23 @@ export class Extensions extends React.Component {
         });
 
         this.addedInstalling.forEach(({ id, displayName }) => {
+          const extension = this.extensions.find(extension => extension.id === id);
+
+          if (!extension) {
+            throw new Error("Extension not found");
+          }
+
           Notifications.ok(
             <p>Extension <b>{displayName}</b> successfully installed!</p>
           );
           this.extensionState.delete(id);
           this.installPath = "";
+
+          // Enable installed extensions by default.
+          // Timeout is required since we're modifying the observable inside reaction.
+          setTimeout(() => {
```

Comment:
> // Timeout is required since we're modifying the observable inside reaction.

Sounds weird.. Observables could/should be modified from anywhere.. 
Maybe you need `runInAction` or mark a method with `@action` when updating something?

Reply:
It's definitely weird, and it's a bit difficult to track what's going on with all the various reactions and computed extensions and store interaction.

I tried `runInAction` and  `@action` but those didn't work.

Reply:
Digging deeper into this one, it seems that the extensions-store will disable it if done sync in the reaction. I'm trying to find a better solution.

Reply:
I removed the timeout but also had to remove the reaction to update state from store updates in extensions-store. I didn't find a reason for that reaction, and enabled state seems to work fine with it removed.

Reply:
This should be fixed now.